src/bindings/python/src/pyopenvino/graph/node_output.cpp

```diff
@@ -13,3 +13,12 @@ namespace py = pybind11;
 
 template void regclass_graph_Output<ov::Node>(py::module m, std::string typestring);
 template void regclass_graph_Output<const ov::Node>(py::module m, std::string typestring);
+
+template <>
+void reg_non_const_methods(output_t<ov::Node>& output) {
+    output.def("replace",
```

Comment:
This method has just been exposed to Python in another PR https://github.com/openvinotoolkit/openvino/blob/master/src/bindings/python/src/pyopenvino/graph/node_output.cpp#L34

I think you can remove it.